src/shadowbox/server/manager_metrics.ts

```diff
@@ -38,4 +41,18 @@ export class PrometheusManagerMetrics implements ManagerMetrics {
     }
     return {bytesTransferredByUserId: usage};
   }
+
+  async getOutboundByteTransfer(accessKeyId: string, windowHours: number):
```

Comment:
I talked to @cjhenck about this and we agreed to measure outbound traffic, since that is what the admin pays and generally counts towards the server's data usage.

> This is one thing that we may want Annabell to answer.

+1